tutor/templates/apps/openedx/settings/partials/common_all.py

```diff
@@ -34,12 +34,14 @@
 # Behave like memcache when it comes to connection errors
 DJANGO_REDIS_IGNORE_EXCEPTIONS = True
 
-# Elasticsearch connection parameters
-ELASTIC_SEARCH_CONFIG = [{
-  {% if ELASTICSEARCH_SCHEME == "https" %}"use_ssl": True,{% endif %}
-  "host": "{{ ELASTICSEARCH_HOST }}",
-  "port": {{ ELASTICSEARCH_PORT }},
-}]
+# Meilisearch connection parameters
+MEILISEARCH_ENABLED = True
+MEILISEARCH_URL = "{{ MEILISEARCH_URL }}"
+MEILISEARCH_PUBLIC_URL = "{{ MEILISEARCH_PUBLIC_URL }}"
```

Comment:
This variable overrides the `MEILISEARCH_PUBLIC_URL` set above in [development.py](https://github.com/overhangio/tutor/pull/1141/files#diff-74ef659bea171686911a1025aa77a9a83fe9f602a3f3fcf201a19c37d8fb2380R18), so tutor dev meilisearch's public URL is missing the `:7700` port.

So I think it should be removed from here, and explicitly set in the lms + cms `production.py` like it is in `development.py`.

```suggestion
```

Reply:
Are you sure? The development.py settings are supposed to override the settings from common_all.py:

```
$ tutor dev run --no-deps lms ./manage.py lms shell -c "from django.conf import settings; print(settings.MEILISEARCH_PUBLIC_URL)"
http://meilisearch.local.openedx.io:7700
```

Reply:
Ah.. you are correct, the lms/development.py settings are fine, because they [import common_lms.py *before* setting MEILISEARCH_PUBLIC_URL](https://github.com/overhangio/tutor/pull/1141/files#diff-74ef659bea171686911a1025aa77a9a83fe9f602a3f3fcf201a19c37d8fb2380R5).

But the cms/development.py [imports common_cms.py *after* setting `MEILISEARCH_PUBLIC_URL`](https://github.com/overhangio/tutor/pull/1141/files#diff-d0396341985a7618fe5e749e2e8f7da3fadea6bb9aec6aefb3e456ac1ff99f7eR19), and so `MEILISEARCH_PUBLIC_URL` gets overwritten.

Can we move the common_cms.py import above?

Reply:
Very good catch. I have no idea how the import found its way so far down. I'll move it back up.